app/javascript/components/enrollment/steps/Contact.js

```diff
@@ -239,67 +239,50 @@ class Contact extends React.Component {
-                        <br />
-                        <span className="font-weight-light">Between 12:00 and 16:00 in monitoree&apos;s timezone</span>
-                        <br />
-                        <span className="font-weight-light">Between 16:00 and 20:00 in monitoree&apos;s timezone</span>
-                      </Form.Group>
-                    </Form.Row>
+                    <div className="my-1">
+                      Morning: <span className="font-weight-light">Between 8:00 and 12:00 in monitoree&apos;s timezone</span>
+                      <br />
+                      Afternoon: <span className="font-weight-light">Between 12:00 and 16:00 in monitoree&apos;s timezone</span>
+                      <br />
+                      Evening: <span className="font-weight-light">Between 16:00 and 20:00 in monitoree&apos;s timezone</span>
+                    </div>
                     <Form.Control.Feedback className="d-block" type="invalid">
                       {this.state.errors['preferred_contact_time']}
                     </Form.Control.Feedback>
                   </Form.Group>
                 )}
               </Form.Row>
               <Form.Row>
-                <Form.Group as={Col} md="11">
-                  <Form.Row>
-                    <Form.Group as={Col}>
-                      <Form.Label htmlFor="primary_telephone" className="input-label">
-                        PRIMARY TELEPHONE NUMBER{schema?.fields?.primary_telephone?._exclusive?.required && ' *'}
-                      </Form.Label>
-                      <PhoneInput
-                        id="primary_telephone"
-                        value={this.state.current.patient.primary_telephone}
-                        onChange={this.handleChange}
-                        isInvalid={!!this.state.errors['primary_telephone']}
-                      />
-                    </Form.Group>
-                    <Form.Group as={Col}>
-                      {this.state.current.blocked_sms && (
-                        <Form.Label className="tooltip-whitespace input-label font-weight-bold py-2">
-                          SMS Blocked <InfoTooltip tooltipTextKey="blockedSMS" location="top"></InfoTooltip>
-                        </Form.Label>
-                      )}
-                    </Form.Group>
-                  </Form.Row>
-                  {this.state.current.patient?.preferred_contact_method?.includes('SMS') && this.state.current.blocked_sms === true && (
-                    <Form.Label className="tooltip-whitespace">
+                <Form.Group as={Col} lg="12" xl="10" controlId="primary_telephone">
+                  <Form.Label className="input-label">PRIMARY TELEPHONE NUMBER{schema?.fields?.primary_telephone?._exclusive?.required && ' *'}</Form.Label>
+                  {this.state.current.blocked_sms && (
+                    <span className="float-right font-weight-bold">
```

Comment:
thoughts about making the font color red like it is on the patient details/review page? since this is a relatively severe warning message, it would be good if it could stand out more in some way and I think the way it's done on the patient page is a decent approach

![Screen Shot 2021-06-16 at 4 58 26 PM](https://user-images.githubusercontent.com/9956561/122292543-17044600-cec4-11eb-9b0a-a0f5eacc7faf.png)

![Screen Shot 2021-06-16 at 4 59 36 PM](https://user-images.githubusercontent.com/9956561/122292686-3e5b1300-cec4-11eb-8ef8-abca1505b322.png)

Reply:
Good suggestion! PO team decided on this:

![Screen Shot 2021-06-17 at 4 09 44 PM](https://user-images.githubusercontent.com/35042815/122465915-a40fd380-cf86-11eb-8fa2-b30e9787c76e.png)

Added in: 
https://github.com/SaraAlert/SaraAlert/pull/941/commits/f01da8ad69956a6e768193931a3b581dc22f570f

Reply:
didn't Becky want the alert color to be yellow? this would also make it consistent with the warning from asymp enrollment https://mitrecorp.slack.com/archives/C01A6P94R9B/p1623959251099200?thread_ts=1623955769.096600&cid=C01A6P94R9B

Reply:
that's for the one I am about to add for the languages warning, haven't pushed yet

Reply:
done: https://github.com/SaraAlert/SaraAlert/pull/941/commits/f22e03c2a896c21fe594c35fa6d6a69f2b8083ed, should be good now

Reply:
This is what they look like:

![Screen Shot 2021-06-17 at 5 19 09 PM](https://user-images.githubusercontent.com/35042815/122473748-454f5780-cf90-11eb-859c-d5cdf86a0d0b.png)

![Screen Shot 2021-06-17 at 5 18 59 PM](https://user-images.githubusercontent.com/35042815/122473750-45e7ee00-cf90-11eb-8d85-8576e829f498.png)